examples/html-preview/README.md

```diff
@@ -1,50 +1,118 @@
-       - `boards:read`
-       - `boards:write`
-4. Open the [`.env`](.env) file, and enter the app client ID and client secret
-   values that you saved at the beginning of step 3 above.
-5. Run `npm start` to start developing.
-
-When your server is up and running:
-
-- Go to [Miro.com](https://miro.com).
-- In your developer team, open a board.
-- To start your app, click the app icon in the app toolbar on the left.
+# Miro HTML Preview App
+
+This app shows how to use the Miro REST API to generate HTML pages from Miro boards. This enables creating multi-page website mockups quickly.
+
+# 👨🏻‍💻 App Demo
+
+https://github.com/miroapp/app-examples/assets/10428517/6b3c90d6-06f0-4dc7-adb3-b7b3c115da84
+
+> The initial screen prompts you to connect your Miro account. After signing in, you can see the list of boards under your team.
+> To view a specific frame, use its title in the URL. In the app, URLs are in the following format: /{boardId}/{frameTitle}.
+
+# 📒 Table of Contents
+
+- [Included Features](#features)
+- [Tools and Technologies](#tools)
+- [Prerequisites](#prerequisites)
+- [Run the app locally](#run)
+- [Folder Structure](#folder)
+- [Contributing](#contributing)
+- [License](#license)
+
+# ⚙️ Included Features <a name="features"></a>
+
+- [Miro Node.js Client](https://developers.miro.com/docs/web-sdk-reference)
+  - [api.getAllBoards](https://miroapp.github.io/api-clients/classes/index._internal_.Api.html#getAllBoards)
+  - [board.getAllItems()](https://miroapp.github.io/api-clients/classes/index.Board.html#getAllItems)
+  - [board.getAllConnectors()](https://miroapp.github.io/api-clients/classes/index.Board.html#getAllConnectors)
+  - [board.getShapeItem()](https://miroapp.github.io/api-clients/classes/index.Board.html#getShapeItem)
+  - [board.getTextItem](https://miroapp.github.io/api-clients/classes/index.Board.html#getTextItem)
+  - [board.getStickyNoteItem](https://miroapp.github.io/api-clients/classes/index.Board.html#getStickyNoteItem)
+  - [board.getItem](https://miroapp.github.io/api-clients/classes/index.Board.html#getItem)
+
+# 🛠️ Tools and Technologies <a name="tools"></a>
+
+- [React](https://react.dev/)
+- [TypeScript](https://www.typescriptlang.org/)
+- [Next.js](https://nextjs.org/)
+
+# ✅ Prerequisites <a name="prerequisites"></a>
+
+- You have a [Miro account](https://miro.com/signup/).
+- You're [signed in to Miro](https://miro.com/login/).
+- Your Miro account has a [Developer team](https://developers.miro.com/docs/create-a-developer-team).
+- Your development environment includes [Node.js 14.13](https://nodejs.org/en/download) or a later version.
+- All examples use `npm` as a package manager and `npx` as a package runner.
+
+# 🏃🏽‍♂️ Run the app locally <a name="run"></a>
+
+1. Rename the `.sample.env` file to `.env` and then add in your environmental variables. Once completed your `.env` file should
+   look something like this:
+
+```
+MIRO_CLIENT_ID=34587644256670555909
```

Comment:
Is this a real ID/Secret? 😨

Reply:
If so: please delete the app this belongs to. We've had bug bounty hunters that we had to pay because of this (May have been my fault)

Reply:
No it's not :)

Reply:
I deleted half the numbers in it and then randomly generated half of the characters in both the client + secret